src/main.js

```diff
@@ -134,10 +137,19 @@ let load = base.loader({
       let regions = cfg.backend.s3.regions.split(',');
 
       for (let region of regions) {
-        let bucket = cfg.backend.s3.bucketBase + cfg.server.env + '-' + region;
+        let bucket;
+        if (profile === 'test') {
+          if (!testBucket) {
+            testBucket = uuid.v4().replace(/-/g, '');
+          }
```

Comment:
Is this really necessary? I'm very confused... do you create buckets now?
Why not just hardcode a name to use for tests?

Note: we can't have as many s3 buckets as we would like... So I wouldn't recommend creating one for each test...
We do have a garbage one for testing...